src/hearings/services/lov-ref-data.service.spec.ts

```diff
@@ -6,8 +6,8 @@ describe('Lov RefData service', () => {
   describe('LovRefDataService', () => {
     it('getForService should make correct api call', () => {
       const service = new LovRefDataService(mockHttpService);
-      service.getListOfValues(HearingCategory.Priority, 'SSCS');
-      expect(mockHttpService.get).toHaveBeenCalledWith(`api/prd/lov/getLovRefData?category=Priority&service=SSCS`);
+      service.getListOfValues(HearingCategory.Priority, 'SSCS', false);
+      expect(mockHttpService.get).toHaveBeenCalledWith(`api/prd/lov/getLovRefData`, jasmine.any(Object));
```

Comment:
is that possible to narrow this to category=Priority&service=SSCS rather than any Obj?

Reply:
as discussed `jasmine.objectContaining` isn't working here as expected.